app/components/dashboard/Dashboard.jsx

```diff
@@ -1,6 +1,14 @@
 import React, { PropTypes } from 'react';
 
-const Dashboard = ({ filters, formulae }) => (
+const Dashboard = ({
+  filters,
+  formulae,
+  selectedFilter,
+  selectedFormula,
+  onSelectFilter,
+  onSelectFormula,
+  onClickConfigure,
```

Comment:
its all being used. when the formula is being selected it is sending the props to the sidebar to render the specific content of the formula. When a filter is clicked we need to send a request to server to recieve the formulas pretaining to that filter. We need to store what is being selected so so that we can give it a class for css.